### PR TITLE
maDLC single point tracking

### LINKED ISSUE #1552: single point tracking in maDLC
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hi, guys! I have a data collection with several animal, but I want to recognize a single point in each animal (e.g the head). Everything works well until start training. It suddenly closes everything. Does anyone had the same experience? In other versions of DLC I was able to label and train a single point in the animal, but this is the first time with the last version.

### Operating System

WIN 10

### DeepLabCut version

2.2.0.1

### DeepLabCut mode

multi animal

### Device type

gpu

### Steps To Reproduce

_No response_

### Relevant log output

_No response_

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by jeylau**
We leverage our current box tracker for single point tracking; the implementation was straightforward
after realizing that detections + `boundingboxslack` do the trick.

It's tested and working, but I'll leave it as a draft so we can discuss where to best add it to the docs.

Fixes #1552

### CODE REVIEW

**Comment by MMathisLab** [comment]
Hey Tabet @ehsainit - would be also happy for your review if you have the time - i.e., does this mean your needs?

@jeylau looks good to me after merge conflicts, thanks!

File: deeplabcut/pose_estimation_tensorflow/predict_videos.py [hunk 1]
```diff
@@ -1297,6 +1298,10 @@ def convert_detections2tracklets(
         )
 
     cfg = auxiliaryfunctions.read_config(config)
+    if len(cfg["multianimalbodyparts"]) == 1 and track_method != "box":
+        warnings.warn("Switching to `box` tracker for single point tracking...")
+        track_method = "box"
```
Comment (MMathisLab) [inline deeplabcut/pose_estimation_tensorflow/predict_videos.py:1303]: maybe this conflicts with pending merge of track goes to config file?

Comment (jeylau) [reply deeplabcut/pose_estimation_tensorflow/predict_videos.py:1303]: Yeah, let's fix it in the other PR (which we'll merge afterwards)

Comment (ehsainit) [reply deeplabcut/pose_estimation_tensorflow/predict_videos.py:1303]: @jeylau Thanks for your work !

Hey @MMathisLab  ill review it by tomorrow and let you know.

**Comment by ehsainit** [comment]
@jeylau Thanks for your work !

Hey @MMathisLab ill review it by tomorrow and let you know.

**Comment by ehsainit** [comment]
> Hey Tabet @ehsainit - would be also happy for your review if you have the time - i.e., does this mean your needs?
> 
> @jeylau looks good to me after merge conflicts, thanks!

Hey @MMathisLab, looks good !

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**